rspirv/dr/constructs.rs

```diff
@@ -34,8 +34,12 @@ pub struct Module {
     pub entry_points: Vec<Instruction>,
     /// All execution mode declarations, using OpExecutionMode.
     pub execution_modes: Vec<Instruction>,
-    /// All non-location debug instructions.
-    pub debugs: Vec<Instruction>,
+    /// Debug subsection: All OpString, OpSourceExtension, OpSource, and OpSourceContinued.
+    pub debug_string_source: Vec<Instruction>,
+    /// Debug subsection: All OpName and all OpMemberName.
+    pub debug_names: Vec<Instruction>,
+    /// Debug subsection: All OpModuleProcessed instructions.
+    pub debug_module_processed: Vec<Instruction>,
```

Comment:
Tiny nit pick: looks like all the other names are pluralized, so I'd expect `debug_string_sources` and `debug_module_processeds`.